addons/mrp/tests/test_order.py

```diff
@@ -1168,17 +1168,17 @@ def test_product_produce_uom(self):
 
         # produce product
         mo_form = Form(mo)
-        mo_form.qty_producing = 1/12.0
         mo_form.lot_producing_id = final_product_lot
+        mo_form.qty_producing = 1/12.0
         mo = mo_form.save()
 
         move_line_raw = mo.move_raw_ids.mapped('move_line_ids').filtered(lambda m: m.qty_done)
-        self.assertEqual(move_line_raw.qty_done, 1)
+        self.assertEqual(move_line_raw.qty_done, 0.96)
```

Comment:
Maybe it is more appropriate to remove this test ?
in case of serial tracking, the quantity won't be equals to 1 UoM which doesn't make any sense.
Maybe force the move line to be in the UoM of product in case of Serial ?

Reply:
ok to remove the test. 
ok to set the product uom on the move line but it seems to be a complete task on it's own

Reply:
@ryv-odoo @Whenrow Anyone on this?

Reply:
A task-2326479 is created for this issue.